python/plugins/db_manager/db_plugins/oracle/info_model.py

```diff
@@ -512,8 +511,7 @@ def getMViewInfo(self):
                     values[9]))
         tbl.append((QApplication.translate(
             "DBManagerPlugin",
-            (u'<a href="action:mview/refresh">Refresh the materializ'
-             u'ed view</a>')),
+            (u'<a href="action:mview/refresh">Refresh the materialized view</a>')),
```

Comment:
Wouldn't it be cleaner to separate the translated part of this string from the html formatting surrounding it?

Reply:
That would result in something like this? Correct?
```py
        tbl.append((u'<a href="action:mview/refresh">{0}</a>'.format(
            QApplication.translate("DBManagerPlugin", "Refresh the materialized view")),
            u""))
```
For information, I couldn't find this string neither in transifex nor qgis_en.ts file 🤔 
![image](https://user-images.githubusercontent.com/7983394/200405194-3ec901ff-9885-4f39-9211-6b9dee0d8b45.png)

Reply:
Yes, that's correct. We don't need the "u" prefixes anymore either.